### ISSUE TITLE
The 'home' button is selected in 'Site Setup'

### ISSUE THREAD

**Issue by jaroel** [plone/Products.CMFPlone#701]
So it looks like:
![screen shot 2015-07-16 at 20 19 08](https://cloud.githubusercontent.com/assets/70700/8731490/002f520a-2bf8-11e5-9766-b0ef69f1e853.png)

but I'd expect it to be like:
![screen shot 2015-07-16 at 20 19 28](https://cloud.githubusercontent.com/assets/70700/8731498/0a555d1a-2bf8-11e5-8d25-87f3e87c81a7.png)

**Comment by vangheem** [comment]
Duplicate of https://github.com/plone/Products.CMFPlone/issues/700 right? Closing.

**Comment by jaroel** [comment]
Nope, it's about the selected state of the homebutton, when we're in the site setup pages.

**Comment by idgserpro** [comment]
This seems to be a problem with every url that begins with @@ at the root of your site. You can check this by going through all options in control panel, or even just adding @@manage-portlets.

This comes from plone.app.layout, that creates the breadcrumb view in sections.pt, this issue should be there. Don't have time to check it and can't do a PR but I think you need to change this line https://github.com/plone/plone.app.layout/blob/cc48b1eea56ce38bd95cb1978043903bc37cd9ef/plone/app/layout/viewlets/common.py#L242, checking if you're into a browser view that begins with '@@' at root and just `return {'portal': ''}`. Don't know if it's the better approach though.

**Comment by davisagli** [comment]
Still valid
